cf_remote/commands.py

```diff
@@ -485,6 +509,31 @@ def init_cloud_config():
     return 0
 
 
+def save(name, hosts, role):
+    state = read_json(CLOUD_STATE_FPATH)
+    if not state:
+        state = {}
+    if "@" + name in state:
+        print("Group '{}' already exists".format(name))
+        return 1
+    group = {}
+    group["meta"] = {}
+    for index, host in enumerate(hosts):
+        split = host.split("@")
+        if len(split) != 2:
+            print("Host '{}' not accepted, must be given as user@ip-address".format(host))
+            return 1
+        user, ip = host.split("@")
+        instance = {}
+        instance["public_ips"] = [ip]
+        instance["user"] = user
+        instance["role"] = role
```

Comment:
I'd create the dictionary in one step instead of creating it and then populating it, but YMMV.

Reply:
Oki.